extensions/wikia/Email/Controller/PasswordResetLinkController.class.php

```diff
@@ -95,4 +91,19 @@ protected function getSummary() {
 	protected function getIntro() {
 		return $this->getMessage( 'emailext-password-reset-link-intro' )->text();
 	}
+
+	private function getResetURL() {
+		global $wgDevelEnvironment;
+
+		if ( !empty( $wgDevelEnvironment ) && !empty( $this->returnUrl ) ) {
```

Comment:
Checking for empty without checking for true always makes me nervous in such context: https://github.com/Wikia/config/blob/a38480fd38821d7a09aa85c130a4e088500b042a/LocalSettings.php#L12 :)